image_width: 1920
image_height: 1080
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [1307.664594,    0.     ,  967.699491,
            0.     , 1303.758503,  578.890043,
            0.     ,    0.     ,    1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.364727, 0.106777, 0.001452, -0.000703, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [ 980.123367,    0.     ,  969.024499 ,    0.     ,
            0.     , 1212.981447 ,  586.377127,    0.     ,
            0.     ,    0.     ,    1.     ,    0.     ]
